.travis.yml

```diff
@@ -50,14 +50,6 @@ matrix:
                CONDA_DEPENDENCIES=$CONDA_ALL_DEPENDENCIES
                PIP_DEPENDENCIES='jplephem'
 
-        # Check for sphinx doc build warnings - we do this first because it
```

Comment:
The commit changing this (with commit message "DEBUG: ..." added by me) should be removed with a rebase.